code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -209,6 +209,7 @@ static int WheelTractionVectorLengthOffset = 0x1B8;
 static int WheelSteeringAngleOffset = 0x1CC;
 static int WheelBrakePressureOffset = 0x1D0;
 static int WheelHealthOffset = 0x1E8; // 75 24 F3 0F 10 81 ? ? ? ? F3 0F
+static int WheelSurfaceMaterialOffset = (Is2060() ? 0x20A : 0x1FE); // 2060 48 8B 4A 10 0F 28 CF F3 0F 59 05 ? ? ? ? - 0x4
```

Comment:
honestly by now just get yourself the bonus points, this is not even guaranteed to work safely

Reply:
give me the bonus points now thanks